### ISSUE TITLE
Please add jni support CMakeList.txt

### ISSUE THREAD

**Issue by halloweeks** [Sketchware-Pro/Sketchware-Pro#622]

**Comment by ilyassesalama** [comment]
I don't think this is a necessary feature since it targets the minority of Sketchware Pro users. However, I don't know, it depends on modders and how difficult the feature is to implement; but IMO, this is unnecessary.
